tests/devapps/NetCoreTestApp/Program.cs

```diff
@@ -54,7 +55,7 @@ public static void Main(string[] args)
             var ccaSettings = ConfidentialAppSettings.GetSettings(Cloud.Public);
```

Comment:
Let's not invent anything new either here. Let's use [MicrosoftAuthenticationOptions](https://github.com/AzureAD/microsoft-identity-abstractions-for-dotnet/blob/main/src/Microsoft.Identity.Abstractions/ApplicationOptions/MicrosoftAuthenticationOptions.cs)